### ISSUE TITLE
Add_ons not populating

### ISSUE THREAD

**Issue by clandorsdx** [Manuel83/craftbeerpi3#226]
In 3.0 when I go to the add_ons tab under system the plugins don't populate.  I saw that there was an issue with this earlier but it was marked as solved.

**Comment by msfuture** [comment]
Same her... fresh clone from git, no addons are loaded. The javascript console prints a 500 Internal Error while loading the plugin list?

**Comment by HooTsBrooklyn** [comment]
Does your rpi have internet access?  The plugins list is populated by a file on github.

**Comment by msfuture** [comment]
Yes of course. git pull worked like a charm, and the version information page is pulling data as well.

**Comment by bitterroots** [comment]
Same here. I have reinstalled and cleared data multiple times.

**Comment by ShintoTuna** [comment]
i resolved this issue by reinstalling my Raspbian OS to Jessie version. it seems that Stretch has issues.
